### PR TITLE
Responsive graph in capture example

### PR DESCRIPTION
**Pull request by symt**
In response to #110 (for gci). 

Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- see README.md for how to run them
* [X] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [X] pull request are descriptively named
* [X] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

<hr>

What it looks like: https://imgur.com/a/l8zFo9M
I'm unsure if that is the correct "item" to be looking at for the change. If it isn't, please tell me and I'll modify the query to fit the correct standard.

### CODE REVIEW

**Comment by chen-robert** [comment]
Welcome to Public Labs @symt!

If you haven't already, be sure to check out the welcoming post at https://github.com/publiclab/plots2/issues/6808 :smile: You can also come to chat with us [on gitter](https://gitter.im/publiclab/publiclab)! 

The changes look great, should be ready to merge now! 
cc: @sidntrivedi012 @jywarren 

Glad to have you here :tada:

File: examples/capture/capture.css [hunk 1]
```diff
@@ -42,3 +42,9 @@ hr {
   height: 200px;
   margin: 10px 0px;
 }
+
+@media only screen and (max-width: 1300px) {
```
Comment (starkblaze01) [inline examples/capture/capture.css]: Hi, @symt Thanks for opening the PR. Things are great so far but you have to add some changes. You have to set different widths for different standard screen sizes. And add gifs for it. Please so open the developer console in the gifs to make sure things are working fine. You can screen sizes for different ranges. Like for up to `max-width: 480px`, `max-width: 992px` and `max-width: 1382px`

Comment (symt) [reply examples/capture/capture.css]: Is something like [this](https://imgur.com/a/hmGR7en) what you're looking for? I'm not sure if the CORS error is relevant here.

Comment (jywarren) [reply examples/capture/capture.css]: Is it possible to use Bootstrap layout classes to manage this in a standarized way? 

https://getbootstrap.com/docs/4.3/layout/grid/

Comment (starkblaze01) [reply examples/capture/capture.css]: Hi, @symt changes look great. Can you add gifs showing different screen sizes on the browser?

Comment (symt) [reply examples/capture/capture.css]: [Here](https://imgur.com/a/jj5n9MZ)

**Comment by jywarren** [comment]
Super! Great work!!! Thank you!

**Pull request merged**